cmd/ocm/login/cmd.go

```diff
@@ -153,6 +163,15 @@ func run(cmd *cobra.Command, argv []string) error {
 		return fmt.Errorf("Option '--url' is mandatory")
 	}
 
+	if args.oauthLogin {
+		fmt.Println("You will now be redirected to Red Hat SSO login")
+		token, err := authentication.VerifyLogin(oauthClientID)
+		if err != nil {
+			return err
+		}
+		args.token = token
```

Comment:
I know we do this in the browser already, but until we get some more UX direction let's include a success message in the CLI as well.

Reply:
Added the message